frontend/src/components/dashboard/aliases/CustomAddressGenerationModal.module.scss

```diff
@@ -0,0 +1,336 @@
+      }
+    }
+
+    .form-wrapper {
+      text-align: center;
+
+      .form-heading {
+        color: $color-dark-gray-70;
+        padding: $spacing-md 0;
+
+        @media screen and #{$mq-sm} {
+          padding: $spacing-lg 0;
+        }
+      }
+
+      .prefix {
+        font-family: $font-stack-firefox;
+        color: $color-grey-50;
+        width: 70%;
+        margin: 0 auto;
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+
+        label {
+          text-align: start;
+          font-weight: 600;
+          margin-top: $spacing-xl;
+        }
+
+        .profile-registered-domain-value {
+          color: $color-blue-50;
+          word-break: break-word;
+          margin-top: $spacing-md;
+          margin-bottom: $spacing-2xl;
+        }
+
+        input {
+          @include form-input;
+          // Override form-input's margin:
+          margin: $spacing-sm 0 0 0;
+          font-weight: 400;
+          padding: $spacing-sm;
+          color: $color-grey-40;
+          font-family: $font-stack-base;
+
+          @include text-body-sm;
+
+          &.invalid-prefix:focus {
+            outline: 5px solid $color-red-10;
+            border: 2px solid $color-error;
+          }
+        }
+
+        input:focus {
+          outline: 5px solid $color-blue-100;
+          border: 2px solid $color-blue-50;
+
+          + .wrapper {
+            display: initial;
+          }
+        }
+
+        .wrapper {
+          position: relative;
+          display: none;
+
+          .errors {
+            display: flex;
+            text-align: left;
+            align-items: center;
+            margin: 0 0 $spacing-sm $spacing-sm;
+            gap: $spacing-md;
+
+            .error-icons {
+              min-width: $spacing-md;
+              flex: 0 0 auto;
+            }
+
+            .check-icon {
+              color: $color-green-80;
+              width: calc($spacing-md - $spacing-xs);
+              height: calc($spacing-md - $spacing-xs);
+            }
+
+            .close-icon {
+              color: $color-error;
+            }
+          }
+
+          .tooltip {
+            display: initial;
+
+            @include text-body-sm;
+            $triangleWidth: 10px;
+            position: absolute;
+            background-color: $color-white;
+            box-shadow: $box-shadow-sm;
+            border-radius: $border-radius-sm;
+            padding: $spacing-md;
+            width: 100%;
+            margin-top: $triangleWidth;
+            left: 0;
+            // Overlap .banner in <Banner>, which is relatively positioned:
+            z-index: 1;
+
+            &:after {
+              background-color: $color-white;
+
+              content: "\00a0";
+              display: block;
+              width: $triangleWidth;
+              height: $triangleWidth;
+              position: absolute;
+
+              top: calc(-1 * $triangleWidth / 2);
```

Comment:
Suggestion (non-blocking): Perhaps it might be helpful for future us to have a comment here as to why we are doing this calculation?

Reply:
Done.